### PR TITLE
Fix remoteOptions field initial state

### PR DESCRIPTION
**Pull request by reupen**
Fixes the initial state of form fields using remoteOptions

### CODE REVIEW

**Comment by reupen** [comment]
Updated to enhance remoteOptions:
* Can now specify some parameters to pass to the API
* Can now use a format string for the option text

File: lux/extensions/rest/models.py [hunk 1]
```diff
@@ -138,9 +140,11 @@ def field_options(self, request):
             target = self.get_target(request)
             yield 'data-remote-options', json.dumps(target)
             yield 'data-remote-options-id', self.id_field
-            yield 'data-remote-options-value', self.repr_field
+            yield 'data-remote-options-value', json.dumps({
+                'type': 'field',
+                'source': self.repr_field})
             yield 'data-ng-options', self.remote_options_str.format(
-                id=self.id_field, repr=self.repr_field, options=self.api_name)
```
Comment (lsbardel) [inline lux/extensions/rest/models.py]: why are you using 'id' and 'name'?

Comment (reupen) [inline lux/extensions/rest/models.py]: see the changes to `luxjs/forms/utils.js`

Comment (lsbardel) [inline lux/extensions/rest/models.py]: I still don't understand, in any case why are you formatting constants?

Comment (reupen) [inline lux/extensions/rest/models.py]: No particular reason, I just went with leaving the replacement fields in remote_options_str. I've changed it now.

**Comment by lsbardel** [comment]
merged, I'll take a closer look, thanks

**Pull request merged**